backend/data/static/data_cleaning_functions.py

```diff
@@ -0,0 +1,255 @@
+# import packages
```

Comment:
move in `/backend/preprocess` package

Reply:
Silly question, but how would I import the functions if it is not in the same directory as raw/clean_datasets?